src/renderer/splash.tsx

```diff
@@ -18,7 +19,7 @@ const Splash = memo(() => {
     useEffect(() => {
         ipcRenderer.on('splash-setup-progress', (event, progress) => {
             setOverallProgress(progress.totalProgress);
-            setStatusProgress(progress.statusProgress > 0 ? progress.statusProgress : null);
+            setStatusProgress(progress.statusProgress > 0 ? progress.statusProgress / 2 : null);
```

Comment:
Why?

Reply:
oops, i did that before learning about the progress slices